### PR TITLE
sp_load_solrprovider_freeze

### PR DESCRIPTION
**Pull request by carresemata**
Created sp_load_solrprovider_freeze in snowflake:
* Can not be validated because the table dependency show.webfreezer has failed to load. Error: Timestamp '(seconds_since_epoch=1677071887916974800)' is not recognized

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql [hunk 1]
```diff
@@ -0,0 +1,411 @@
+-- hack_spuMAPFreeze
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRPROVIDER_FREEZE()
```
Comment (omargoldpr) [inline migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql:2]: Add the Database to the procesure name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRPROVIDER_FREEZE() 
```

File: migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+                    FFDisplaySpecialty,
+                    FFPESBoost,
+                    FFMalMultiHQ,
+                    FFMalMulti,
+                    CLinicalFocusXML,
+                    ClinicalFocusDCPXML,
+                    SyndicationXML,
+                    TeleHealthXML ';
+
+select_statement := 'WITH CTE_ClientCode AS (SELECT
```
Comment (omargoldpr) [inline migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql:191]: Select in a new line
```suggestion
select_statement := 'WITH CTE_ClientCode AS (
                     SELECT
```

**Review by omargoldpr — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql (hunk 1 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql:2]: Done

File: migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql (hunk 2 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql:191]: Done

**Pull request merged**